scripts/track-dependency-status.sh

```diff
@@ -0,0 +1,48 @@
+# Get total count of python dependencies and outdated python dependencies
+
+total_python_deps="$(pip list| wc -l)"
+outdated_python_deps="$(pip list --outdated| wc -l)"
+
+# Get outdated JS dependency count
+
+COUNT_OUTDATED_DEP_CODE=$(cat <<-END
+import json
+import sys
+
+dep_arr = json.load(sys.stdin)
+print(len(dep_arr["data"]["body"]))
+
+END
+)
+
+outdated_js_deps="$(yarn outdated --json | sed -n 2p | python -c "$COUNT_OUTDATED_DEP_CODE")"
```

Comment:
nit: I prefer to "shell-ify" things like this by putting them in functions. Maybe something like this?

```suggestion
function outdated_count {
    python - <(cat -) <<-END
import json
import sys
with open(sys.argv[1], "r") as json_file:
    dep_arr = json.load(json_file)
print(len(dep_arr["data"]["body"]))
END
}

outdated_js_deps="$(yarn outdated --json | sed -n 2p | outdated_count)"
```

Reply:
This could also be simplified if `jq` is installed:

```bash
outdated_js_deps=$(yarn outdated --json | sed -n 2p | jq -r '.data.body[]' | wc -l)
```

Reply:
> I prefer to "shell-ify" things like this by putting them in functions. Maybe something like this?

It looks good. I will update it. Thanks for the suggestion.

I had also thought of using `jq` first but we would need to install it. So that's why went ahead with Python.

Reply:
> So that's why went ahead with Python.

That's what I figured, the Python solution is great in the absence of `jq`.